client/src/components/InfoCard.tsx

```diff
@@ -1,77 +1,90 @@
-import { useState } from "react";
-import {
-  Avatar,
-  Badge,
-  Box,
-  Button,
-  Card,
-  CardBody,
-  CardFooter,
-  CardHeader,
-  ChakraProvider,
-  Flex,
-  Heading,
-  IconButton,
-} from "@chakra-ui/react";
+import { Dispatch, SetStateAction, useState } from "react";
+import { Avatar, Badge, Box, Button, Image, Textarea } from "@chakra-ui/react";
 import { Autocomplete, Option } from "chakra-ui-simple-autocomplete";
 import { BiLike, BiChat, BiShare } from "react-icons/bi";
 import { StarIcon } from "@chakra-ui/icons";
-const InfoCard = () => {
-  const property = {
-    imageUrl: "https://bit.ly/2Z4KKcF",
-    imageAlt: "Rear view of modern home with pool",
-    beds: 3,
-    baths: 2,
-    title: "2445 Guilford dr, Abbotsford BC ",
-    formattedPrice: "$2,900",
-    reviewCount: 20,
-    rating: 4,
+import { Post } from "./interfaces/Post";
+import { Comment } from "./interfaces/Comment";
+import "./../styles/userHome.css";
+
+interface props {
+  post: Post;
+  comment: Comment;
+  setComment: Dispatch<SetStateAction<Comment>>;
+  updateComments: (e: any) => void;
+}
+const InfoCard: React.FC<props> = (props) => {
```

Comment:
I don't think you need to do `React.FC<props>` for every components. Just doing a typed function is enough.

See https://stackoverflow.com/questions/59988667/typescript-react-fcprops-confusion